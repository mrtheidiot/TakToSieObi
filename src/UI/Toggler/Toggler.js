import classes from "./Toggler.module.css";

const Toggler = ({ onChange, current }) => {
  return (
    <label className={classes.switch}>
      <input type="checkbox" onChange={onChange} checked={current} />
      <span className={classes.slider} />
    </label>
  );
};

export default Toggler;
