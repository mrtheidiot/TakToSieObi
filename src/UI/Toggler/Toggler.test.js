import { render } from "@testing-library/react";

import Toggler from "./Toggler";

it("matches the Toggler snapshot", () => {
  const {
    container: { firstChild },
  } = render(<Toggler />);
  
  expect(firstChild).toMatchSnapshot()
});
