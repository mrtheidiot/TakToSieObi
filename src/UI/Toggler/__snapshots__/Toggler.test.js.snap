// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the Toggler snapshot 1`] = `
<label
  class="switch"
>
  <input
    type="checkbox"
  />
  <span
    class="slider"
  />
</label>
`;
